src/backend/distributed/sql/citus--10.0-3--10.0-4.sql

```diff
@@ -0,0 +1,9 @@
+--
+-- 10.0-3--10.0-4 was added later as a patch to fix a bug in our PG upgrade functions
+--
+-- Users who took the 9.5-2--10.0-1 upgrade path already have the fix, but users
```

Comment:
I think this comment is not valid anymore because there is no 9.5.2--10.0.1 path right?

Reply:
there is a valid path for `9.5-2--10.0-1`, it goes via `9.5.1`.
postgres doesn't interpret versions, _luckily_.

users that came from 9.5-2 would still have the fix already installed as we deliberately do not install the broken upgrade function on downgrades.

Reply:
I think I wasn't clear in my question, I meant to ask we don't have direct path from 9.5.2 to 10.0.1(such as we had with Marco's version). This is a minor thing but when I first read it I assumed a direct path, I think we can leave it as is for sure